workitemtype tests failing after updating the type of field definition constants from Byte to String.

Executed 
`go test`
inside models/

```
--- FAIL: TestJsonMarshalListType (0.00s)
	workitemtype_test.go:32: json: error calling MarshalJSON for type models.FieldDefinition: invalid character '%' looking for beginning of value
	workitemtype_test.go:39: not the same type {1 first type map[aListType:{false {{list} {integer}}}]}, {0  map[]}
--- FAIL: TestMarshalEnumType (0.00s)
	workitemtype_test.go:61: json: error calling MarshalJSON for type models.FieldDefinition: invalid character '%' looking for beginning of value
	workitemtype_test.go:67: not the same type: {0  map[]}, {1 first type map[aListType:{true {{enum} {} [open done closed]}}]}
--- FAIL: TestMarshalFieldDef (0.00s)
	workitemtype_test.go:83: json: error calling MarshalJSON for type models.FieldDefinition: invalid character '%' looking for beginning of value
	workitemtype_test.go:89: not the same : {false <nil>}, {true {{enum} {} [open done closed]}}

```

--- a/models/field_definition.go
+++ b/models/field_definition.go
@@ -118,7 +118,7 @@ func (self *FieldDefinition) UnmarshalJSON(bytes []byte) error {
 func (self FieldDefinition) MarshalJSON() ([]byte, error) {
 	var buf bytes.Buffer
 	buf.WriteString("{ \"type\": {")
-	buf.WriteString(fmt.Sprintf("\"kind\": %d", self.Type.GetKind()))
+	buf.WriteString(fmt.Sprintf("\"kind\": \"%s\"", self.Type.GetKind()))
 	switch complexType := self.Type.(type) {
 	case ListType:
 		buf.WriteString(", \"extra\": ")

